lib/l10n-en_GB.js

```diff
@@ -112,28 +118,45 @@ exports.messages = {
 ,   "sotd.pp.no-jan24":          "No valid link to 2002 Jan 24 patent policy (eg absent, or not using HTTPS)."
 ,   "sotd.pp.no-pp-transition":  "No valid link to PP transition procedure (eg absent, or not using HTTPS)."
 ,   "sotd.pp.joint-publication": "Joint publication detected."
 ,   "sotd.pp.expected-pp":       "Expected text related to patent policy requirements: <blockquote>${expected}</blockquote>"
-,   "sotd.pp.deprecated":        "The patent policy link (${policy}) you are using is no longer allowed. Please use the undated link <code>https://www.w3.org/Consortium/Patent-Policy/</code>."
-,   "sotd.pp.deprecatedAllowed": "The patent policy (${policy}) you are using is deprecated. Please use the undated link <code>https://www.w3.org/Consortium/Patent-Policy/</code>."
 ,   "sotd.pp.no-errata":         "No errata link was found in the patent policy section."
+,   "sotd.pp.no-response":       "The connection to the API failed: HTTP ${status}"
     // sotd/charter-disclosure
 ,   "sotd.charter-disclosure.no-group":       "Unable to find the deliverer id."
 ,   "sotd.charter-disclosure.text-not-found": "No texts about charter for disclosure obligations found."
 ,   "sotd.charter-disclosure.link-not-found": "No link to charter for disclosure obligations found."
 ,   "sotd.charter-disclosure.wrong-link":     "The link to charter for disclosure obligations should be <a href='${expectedHref}'>${expectedHref}</a>."
 ,   "sotd.charter-disclosure.no-response":    "The connection to the API failed: HTTP ${status}"
+    // sotd/rec-addition
+,   "sotd.rec-addition.no-correction":  "This document contains substantive changes, cannot find corresponding paragraph <code>&lt;p class=\"correction\"&gt;Proposed corrections are marked in the document.&lt;/p&gt;</code>"
+,   "sotd.rec-addition.no-addition":    "This document contains new feature(s), cannot find corresponding paragraph <code>&lt;p class=\"addition\"&gt;Proposed additions are marked in the document.&lt;/p&gt;</code>"
+,   "sotd.rec-addition.wrong-text":     "The text in 'p.${type}' paragraph should be <code>${CORRECTION}</code>"
+,   "sotd.rec-addition.unnecessary-correction": "This document doesn't contain substantive changes, paragraph <code>&lt;p class=\"correction\"&gt;Proposed corrections are marked in the document.&lt;/p&gt;</code> should be removed."
+,   "sotd.rec-addition.unnecessary-addition": "This document doesn't contain new features, paragraph <code>&lt;p class=\"addition\"&gt;Proposed additions are marked in the document.&lt;/p&gt;</code> should be removed."
+    // sotd/publish
+,   "sotd.publish.not-found": "Cannot find the following paragraph in Status of This Document: <code>&lt;p&gt;${txt}&lt;/p&gt;</code>."
+    // sotd/rec-comment-end
+,   "sotd.rec-comment-end.not-found": "This W3C Recommendation has subtantive changes or new features, cannot find comment end date no earlier than ${minimumEndDate}."
```

Comment:
Format the date in a human readable format. Currently, that's the message that is being generated:
"This W3C Recommendation has subtantive changes or new features, cannot find comment end date no earlier than Sat Nov 14 2020 00:00:00 GMT+0000 (Coordinated Universal Time)."

Only "Nov 14 2020" is be enough